chia/plot_sync/receiver.py

```diff
@@ -88,6 +94,9 @@ def current_sync(self) -> Sync:
     def last_sync(self) -> Sync:
         return self._last_sync
 
+    def initial_sync(self) -> bool:
+        return self._last_sync.sync_id == 0
```

Comment:
Already brought up in chat, but for the record here, could we just have `self._last_sync` be `None` instead of having to check an attribute for a special value?

Reply:
Like discussed in chat im not sure yet if its possible but i have it on my list and will revisit it later.